mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
> which refers to a point magnetometer with a normal vector in the z-direction

That's what I'm asking about -- where is it defined in the `z` direction? In this file it looks like it's in the `x` direction, and the comment suggests that, too.